doc/examples/opencv/README.md

```diff
@@ -108,7 +122,7 @@ You can view this data by mounting it locally:
 ```sh
 $ mkdir /tmp/pfs
 $ pachctl mount /tmp/pfs
```

Comment:
Just add `&` to the actual command we list here so usees can copy/paste. 

On a separate note, do we want to them to view the images with FUSE at all? I think opening up the local files in a browser is actually better.